src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectMethod.cs

```diff
@@ -4,7 +4,7 @@
     /// Lists the different authentication methods used to
     /// redirect the user agent to the identity provider.
     /// </summary>
-    public enum OpenIdConnectAuthenticationMethod
+    public enum OpenIdConnectMethod
```

Comment:
OpenIdConnectRedirectBehavior seems fine. 

That said, removing Authentication from all the middleware except cookies is really weird. For consistency and to avoid collisions (GoogleOptions or FacebookOptions are far too generic and may result in duplicate names if you use a FB/Google client in Startup.cs), I'd keep Authentication everywhere.